plugins.xml

```diff
@@ -1,5 +1,24 @@
 <?xml version="1.0"?>
 <plugins xmlns="http://pkp.sfu.ca" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://pkp.sfu.ca plugins.xsd">
+	<plugin category="generic" product="epubJsViewer">
+		<name locale="en_US">EPUB Viewer</name>
+		<homepage>https://github.com/EKT/epubJsViewer-ojs</homepage>
```

Comment:
Thanks, @sioulasdimitris, and sorry for the wait. I've reviewed the code for this plugin in its repository (https://github.com/EKT/epubJsViewer-ojs); nothing major, but a few things to look at!

Reply:
Hello @asmecher,
I appreciate your help.
I have made the changes you asked with some editing.
Please let me know if there is something further you want me to check so I can proceed.
Looking forward to hearing from you.
Thank you in advance

Reply:
Thanks, @sioulasdimitris! The dependencies are included if you build the plugin for release using the PKP plugin build tool (see https://docs.pkp.sfu.ca/dev/plugin-guide/en/release for details).

It looks like the epub.js source is still committed to the repository in the latest commit (https://github.com/EKT/epubJsViewer-ojs/commit/b4680a98eee282a30da0d15e39755446f787e6bf) -- I recommend removing it, so that it can be wrapped up in the `.tar.gz` package only when it's built.

Reply:
Hello @asmecher, thank you for help.
If you check here https://github.com/EKT/epubJsViewer-ojs I believe the files are correct and the epub.js lib is not commited. It is defined as a dependency in package.json file. 
Although I do have  an error with your building command on windows:
Building the release
(node:4332) UnhandledPromiseRejectionWarning: Error: Command failed with ENOENT: mktemp -d
spawn mktemp ENOENT
'mktemp' is not recognized as an internal or external command,

Reply:
@sioulasdimitris, I haven't tried this myself (I'm on Linux), but you should be able to install mktemp from e.g. here: http://gnuwin32.sourceforge.net/packages/mktemp.htm

There may be other *NIX tools/utilities that you also have to install.

Reply:
Thank you @asmecher, this is resolved.
When I execute the command pkp-plugin etc.. I do receive a message that composer.json file is missing. Do i have to use composer as well? 
If you see my package.json file it works file, but when I try to upload the tar.gz file, through ojs plugin gallery the dependencies are not being installed.

Reply:
I have upload the plugin and created a new pull request https://github.com/pkp/plugin-gallery/pull/14

Reply:
@sioulasdimitris I will update the pkp-plugin to run `npm install` if a `package.json` is present. That case wasn't handled (we only dealt with composer and gulp). But for now, what you edit (editing buildRelease.js) should get you going fine

Reply:
I publish version `0.6.1` now, it should handle package.json correctly

Reply:
thank you @kabaros

Reply:
Hello @asmecher,
Please let me know when you are ready, if everything is ok with the plugin in order to merge.
Thank you in advance.